.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false

```

Comment:
Do we need blank issue template?

Reply:
To be organization issue template, i think we can keep blank issue enabled. So that user can use sometimes if the template does not really match against what they are reporting.